integ/lib/testing-tier.ts

```diff
@@ -0,0 +1,268 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as path from 'path';
+import { BastionHostLinux, InstanceType, Port, SubnetType, Vpc } from '@aws-cdk/aws-ec2';
+import { FileSystem } from '@aws-cdk/aws-efs';
+import { ILogGroup } from '@aws-cdk/aws-logs';
+import { Asset } from '@aws-cdk/aws-s3-assets';
+import { CfnOutput, Construct, Duration, Stack, StackProps } from '@aws-cdk/core';
+import { X509CertificatePem, MongoDbInstaller, MongoDbSsplLicenseAcceptance, MongoDbVersion } from 'aws-rfdk';
+import { IWorkerFleet, RenderQueue, Repository } from 'aws-rfdk/deadline';
+import { IRenderFarmDb } from './storage-struct';
+
+// Interface for choosing userData installation options
```

Comment:
It appears `integ` directory is not subject to the same linting rules that enforce a common code style to the rest of the repository code.

While fixing that issue is definitely not in scope for this particular pull request, how would you feel about changing the comments (where applicable) in this file to conform to [TSDoc](https://github.com/microsoft/tsdoc) syntax? This is more important for common code files like this one containing a shared base class since it makes the documentation discoverable in code editors and IDEs that have TypeScript integration. For example, in VS Code, you can hover your mouse over a method and see its TSDoc string in a tooltip. So for example:

```suggestion
/**
 * Interface for choosing userData installation options
 */
```

Reply:
> It appears integ directory is not subject to the same linting rules that enforce a common code style to the rest of the repository code.

Is it? The integ files are included when I run the `build` scripts and it gives the linting errors for missing semicolons, misaligned whitespace, etc. What else is supposed to be included?

Either way I'm find making another pass to clean up those comments.

Reply:
I believe we don't run `awslint` which means some linting is skipped. That linter has a rule to add TSDocs to all public exported classes/methods/interfaces/attribuites.

The [`integ/package.json` uses a different `"build"` script](https://github.com/aws/aws-rfdk/blob/75d0f9fedce52de9dcfe73482965de923fa8941d/integ/package.json#L44). Our main package at [`packages/aws-rfdk/package.json` uses `cdk-build`](https://github.com/aws/aws-rfdk/blob/75d0f9fedce52de9dcfe73482965de923fa8941d/packages/aws-rfdk/package.json#L46) which explains it.

Reply:
Much better. Thanks!